scripts/bash/run_standardization_conformance.sh

```diff
@@ -17,7 +17,7 @@ SRC_DIR=`dirname "$0"`
 
 source ${SRC_DIR}/enceladus_env.sh
 
-export CLASS=${STD_CONF_CLASS}
+export CLASS="za.co.absa.enceladus.standardization_conformance.StandardizationAndConformanceJob"
```

Comment:
I see the move from the `enceladus_env.template.sh` and I support it. However, how could this simple move have helped with the non-working helper script?

Reply:
Interesting. I am actually against pulling variable strings out of env file. This might need changes down the road, maybe even for debug reasons, maybe for some other reasons. As such I would not like this to be in the static part of scripts.

Reply:
> @Zejnilovic The class is not environment specific. It's constant for the job called. This is why I believe it's more then justified to be moved out.

It is not environment-specific, but it feels like it is run specific. Not static. So if you would like to debug an experimental class like HyperConformance, you would need to change more then one place.

Reply:
It's exactly on one place (for each of job types, that is).
And if one wants to do such things, I am not sure  helper script is what one should use. At least not the "main stock" helper scripts. And if fiddling with them in this manner, it imho needs expert knowledge.
I perhaps see what you mean, but then we should create a `encaladus_const.sh` file. IMHO it's better to have this strictly defined somewhere, then to depend on something that is specific for env.